angular09thOct/src/app/view-list/view-list.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, Input } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+@Component({
+  selector: 'app-view-list',
+  standalone: true,
+  imports: [CommonModule],
+  templateUrl: './view-list.component.html',
+  styleUrls: ['./view-list.component.scss']
+})
+export class ViewListComponent {
+  @Input() items: string[] = [];
+
+  deleteItem(index: number) {
+    this.items.splice(index, 1);
+  }
```

Comment:
shouldn't we emit this update ?

Reply:
since the count i am taking is through items.length so it will be updated from the data member items and delete method is in same component where item list is given  ,in the view list component so its working without emiting it directly also